exporter/awsemfexporter/metric_translator.go

```diff
@@ -101,7 +101,7 @@ func (mt metricTranslator) translateOTelToGroupedMetric(rm *pdata.ResourceMetric
 	timestamp := time.Now().UnixNano() / int64(time.Millisecond)
 	var instrumentationLibName string
 	cWNamespace := getNamespace(rm, config.Namespace)
-	logGroup, logStream := getLogInfo(rm, cWNamespace, config)
+	logGroup, logStream, patternReplaceSucceeded := getLogInfo(rm, cWNamespace, config)
```

Comment:
To avoid placeholder pattern check on every single metric data point. Can we set `patternReplaceSucceeded=true` if no `placehoder` pattern(`{`, `}`) found here in either `loggroup` or `logstream` values?

Reply:
Hi Min, if I am not wrong, I am already doing so. If the pattern `{SomeKey}` is not found in any of them (logGroup or logStream), getLogInfo will return true.